lerna.js

```diff
@@ -0,0 +1,21 @@
+const { exec } = require("child_process").exec;
```

Comment:
- you should use a sync variant for simplicity and avoiding possible race conditions in tests.
- You should use `spawnSync` as it returns a stream which can be attached to the parent process.